openedx/core/lib/tempdir.py

```diff
@@ -17,3 +17,22 @@ def cleanup_tempdir(the_dir):
     """Called on process exit to remove a temp directory."""
     if os.path.exists(the_dir):
         shutil.rmtree(the_dir)
+
+
+def mksym_link(src, dest):
```

Comment:
Unless there's a good reason for this operation to be named 'mksym_link', I think it would be better to change the name to 'make_symlink' in order to read more like 'clean_symlink'